### ISSUE TITLE
Add a customizable Date/DateTime field with custom picker

### ISSUE THREAD

**Issue by wdelfuego** [wdelfuego/nova-datetime#3]
Nova 4 defers formatting of Date/DateTime values in pickers to the browser by using its default picker implementation.

Many users have requested ways to customize the picker in different ways.

A custom DateTime field that implements its own customizable DateTime picker would be a super valuable addition to this package.

**Comment by Sotiris-k** [comment]
With the upgrade to Nova 4 I couldn't understand their decision to use the native datepickers. They are incredibly limited.

Nova 3 had flatpickr to do the job if I remember correctly and it was great at it. If this package can bring that back it would just be exactly what's asked for by all these people.

**Comment by Elshaden** [comment]
I Think that Nova 4, should drop the current datepicker, and use a more practical one, since, I am sure, that there is more users who prefer non US date format as default. Using the current one that takes your browser  locale, is not really practical. I for example use en US browser locale but live in Libya. where the en UK date format is used, like everywhere else outside USA. The other problem which is  my clients, who complain about the datepicker showing them US format, most of them are just like myself with US- locale browser but live in the Middle east.

I think it is an important enhancement to Nova, and should be available if not by default , then by a plugin.

**Comment by wdelfuego** [comment]
I agree and I am very open to adding a good, customizable custom DateTime Nova field to this package, I just don't have the time to invent one right now. If you're willing to build one, or at least start the development, or find one that is MIT licensed, give me a heads-up and we'll see if we can get this thing going, shouldn't be too complicated of a feature I think.

**Comment by Elshaden** [comment]
Hi, I suggest Pickaday

https://github.com/Pikaday/Pikaday

it is straight forward and is widely used by Livewire

**Comment by wdelfuego** [comment]
It's a nice date picker but it doesn't look like a candidate for a datetime picker
